nimare/tests/new_json_format/dset_study1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Studies": [
+    {
+      "Groups": [
+        {
+          "@id": "gg_id1",
+          "Name": "group1",
+          "N": 12
+        }
+      ],
+      "Conditions": [
+        {
+          "@id": "cc_id1",
+          "Name": "household items"
+        },
+        {
+          "@id": "cc_id2",
+          "Name": "animals"
+        }
+      ],
+      "@id": 9593960,
+      "Analyses": [
+        {
+          "Images": {
+            "Elements": [
+              {
+                "Path": "orig/9593960_1_beta.nii.gz",
+                "Value": "Beta"
+              },
+              {
+                "Path": "orig/9593960_1_se.nii.gz",
+                "Value": "StandardError"
+              },
+              {
+                "Path": "orig/9593960_1_z.nii.gz",
```

Comment:
What would this be relative to? The JSON file itself? We should probably talk about how to handle file packaging, as this has come up in other places.